​

ItzLevvie opened the pull request:
​

Commit:
Update randomskins.txt

Commit:
Update ClearAll.js

Commit:
Update ClearOwned.js

Commit:
Update UpdatePosition.js

Commit:
Update AddNode.js

Luka967 commented on src/packet/BinaryWriter.js:
```diff
@@ -8,7 +8,7 @@
 
 function BinaryWriter() {
-    this._buffer = new Buffer(1e5);
+    this._buffer = Buffer.allocUnsafe(1e5);
```
[From the nodejs v7.4.0 docs:](https://nodejs.org/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe)
> In versions of Node.js prior to v6, `Buffer` instances were created using the `Buffer` constructor function, which allocates the returned `Buffer` differently based on what arguments are provided:
> ...

This will generate a ReferenceError if node version is below v6.
Even though the current LTS v6.9.4, add a failsafe just in case:
```js
    this.buffer = parseInt(process.version[1]) < 6 ? new Buffer(1e5) : Buffer.allocUnsafe(1e5);
```

ItzLevvie commented:
@Luka967 I added it here https://github.com/ItzLevvie/MultiOgar-Edited/commit/a1101e7ba96e306da5c65f46c0b2791438692ee2 but I reverted I when I was testing

Luka967 commented:
The version is a string not a number xD

ItzLevvie commented:
@Luka967 I know :P my test version has it fixed.

Megabyte918 commented:
Would you like me to commit the fix for you @ItzLevvie ?

ItzLevvie commented:
Merge? @Megabyte918 

Used @Luka967's changes on mobile xD

Luka967 commented:
xD

Commit:
Update BinaryWriter.js

Commit:
Update BinaryReader.js

Megabyte918 commented:
👍
